src/inverse-kinematics/src/ConvexHullHelpers.cpp

```diff
@@ -342,20 +342,31 @@ namespace iDynTree
         return margin;
     }
 
-    Vector2 ConvexHullProjectionConstraint::projectAlongGravity(iDynTree::Position& posIn3dInAbsoluteFrame)
+    void ConvexHullProjectionConstraint::setProjectionAlongDirection(Vector3 direction, const iDynTree::Position originOfPlaneInWorld)
```

Comment:
@aikolina  The `originOfPlaneInWorld` argument seems to be unused.

Reply:
I forgot to add a line:
```C++
o = originOfPlaneInWorld;
```

Reply:
I am a bit confused why you should change the origin of the plane on which you are projecting the point in this call. If this is the case, why don't we set again also `xAxisOfPlaneInWorld` and `yAxisOfPlaneInWorld` ?